azure_billing/scrape.py

```diff
@@ -74,7 +74,7 @@ def data(enrollment, token, month=None):
 
     df = convert_json_df(raw)
 
-    c = Counter('azure_costs', "Costs billed to Azure Enterprise Agreement {}".format(enrollment))
+    c = Counter(metric_name, "Costs billed to Azure Enterprise Agreement {}".format(enrollment))
```

Comment:
So since it is not guaranteed that the units of the values for two different enrolments are the same ($, €), I guess it would be too dangerous to go down the label route and will therefore keep it as it is. The cost unit is not part of the billing API, therefore there is no automated way to prevent different base units.